pyproject.toml

```diff
@@ -72,19 +72,7 @@ dist-dir = "build/dist"
 [tool.ruff]
 target-version = "py37" # Lowest python version supported
 extend-include = ["SConstruct", "SConscript"]
-extend-exclude = [
-    "bench",
-    "bin",
-    "doc",
-    "src",
-    "template",
-    "test",
-    "testing",
-    "timings",
-    "SCons/Tool/docbook/docbook-xsl-1.76.1",
-    "bootstrap.py",
-    "runtest.py",
-]
+extend-exclude = ["SCons/Tool/docbook/docbook-xsl-1.76.1"]
```

Comment:
It's a big pain when the output is cluttered with complaints from those, why remove the exclusion? Except maybe for runtest, for an ordinary run checker run, code that isn't going to be released has little value to be checked (and tests cheat so much that I *really* don't want to see those).

Reply:
Yes, I'd rather add mypy excludes than drop the lint/ruff ones, but it's just one opinion.

Reply:
Trying out a version of it now, and it *works* but I'm not sure if this is the best way to go about it. A blanket ignore for a third-party library that shouldn't be touched is one thing, but ignoring our own code seems better suited for per-user settings

Reply:
on some level you're right, but... not even sure how you set up additional excludes if there's already a version-controlled setup file in place

Reply:
I've been trying to figure that out in VSCode. Ruff was a non-issue, but mypy just refused to cooperate. I suppose keeping the excludes & having them synced is ultimately fine; they can always be trimmed away in dedicated PRs